virttest/qemu_vm.py

```diff
@@ -4315,6 +4315,36 @@ def check_block_locked(self, value):
                     return block['locked']
         return False
 
+    def check_device_tray_opened(self, value):
+        """
+        check whether specified block device is opened or not.
+        Return True, if device is locked, else False.
+
+        :param vm: VM object
+        :param value: block device identifier
+        """
+        assert value, "Device identifier not specified."
```

Comment:
Using `assert` in `avocado-vt` is not a good idea, we should do the check and raise a more accurate exception when check failed (in this case, it would be `ValueError`).